python_modules/dagster/dagster/_serdes/serdes.py

```diff
@@ -368,6 +368,15 @@ def __whitelist_for_serdes(klass: T_Type) -> T_Type:
     return __whitelist_for_serdes
 
 
+def is_serdes_decorated_namedtuple(val: Any, whitelist_map: WhitelistMap = _WHITELIST_MAP) -> bool:
```

Comment:
feel free to ignore, but worth considering the name `is_whitelisted_for_serdes`

Reply:
That name does feel cleaner, but since this method doesn't work for objects other than named tuples I think this needs to be emphasized in the method name

Reply:
enums also can be

Reply:
ah - could it make sense to also return true if it's a whitelisted enum?

Reply:
Or `is_whitelisted_for_serdes_namedtuple` - the main thing is communicating that this is returning true if it's whitelisted for serdes. The decorator is the main way to whitelist something for serdes, but it's just the way it's done, not the outcome.

Reply:
good call -- i renamed to `is_whitelisted_for_serdes_namedtuple`